letterboxdpy/movie.py

```diff
@@ -287,11 +305,11 @@ def movie_watchers(movie: Movie) -> dict:
     dom = dom.find("div", {"id": ["content-nav"]})
 
     data = {
-        'watch_count': 0,
+        'watch_count': movie.watch_count if movie.watch_count else 0,
```

Comment:
You can use the expression `movie.watch_count*bool(movie.watch_count)` for the operations here.